run.py

```diff
@@ -1,34 +1,41 @@
+#!/usr/bin/env python3
+import subprocess
 import time, os, shutil, json
+from subprocess import run
+
+
+def get_battery_percentage():
+    battery_percentage_command = "cat $(find /sys/class/power_supply/ -name 'BAT*' | head -n 1)/capacity"
+    result = run(
+        battery_percentage_command,
+        shell=True,
+        stdout=subprocess.PIPE,
+        universal_newlines=True)
+    return result.stdout.strip()
+
 
 def main():
-    bgImage = ""
+    background_image = ""
     while True:
         # Runs Script that gets current Battery Charge
-        os.system("./battery.sh")
-        time.sleep(1)
-        # Reads The Charge. Note: Looking for a better way to do this
-        with open('data.json', 'r') as f:
-            data = json.load(f)
-            val = int(data["Charge"])
-        a = 100
+        battery_percentage = int(get_battery_percentage())
+        percentage_index = battery_percentage
         # Checks the values
-        while a != 0:
-            if val == a:
-                bgImage = "./values/"+ str(val) +".png"
-                if os.path.isfile(bgImage) != True:
```

Comment:
but what if the image can be found? you are now done, but you continue decreasing a all the way to 1 to exit the while loop, lucky for the val == a else it would lookup a lot of files, but not necessary my friend